src/Client/Facebook.php

```diff
@@ -38,17 +38,17 @@
  */
 final class Facebook extends OAuth2
 {
-    private string $authUrl = 'https://www.facebook.com/dialog/oauth';
-    private string $tokenUrl = 'https://graph.facebook.com/oauth/access_token';
-    private string $endpoint = 'https://graph.facebook.com';
+    protected string $authUrl = 'https://www.facebook.com/dialog/oauth';
```

Comment:
Why "protected"? In final classes "private" equal to "protected".

Reply:
Inheritance?

Reply:
ha-ha, exactly 😀